internal/demonstration.go

```diff
@@ -0,0 +1,21 @@
+package internal
+
+import (
+	"encoding/json"
+	"net/http"
+	"time"
+)
+
+func (h *Handle) Demonstrations(w http.ResponseWriter, r *http.Request) {
```

Comment:
I think demonstration table should be in a normal database like Postgres so that these queries will be fast.
Only big data should be in BigQuery

Reply:
currently there is only 1 table for announcements. this method is handler for the REST API and it calc the count of announcements.
do you think we need to add another table?